### PR TITLE
Update mds

### PR DESCRIPTION
**Pull request by szszeker**
The mds have been updated according to the suggestions of the TC team.

- _The data import process finishes without any immediate signs of error_ section has been reworked.
- "handled by the the Power BI Connector" means the connector wraps the original error in a new error with extra details.

### CODE REVIEW

File: SafeguardSessions/TROUBLESHOOTING.md [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
 ### Collect information
 
-When you import or refresh data with the Power BI Connector, you will likely face one of the scenarios when error occurs:
+When you import or refresh data with the Power BI Connector, you will likely face one of the scenarios when an error occurs:
 
-- The data import finishes without any immediate signs of error.
+- The data import process finishes without any immediate signs of error.
 - A pop-up appears indicating an error.
-- An error occurs when connecting via the Power BI Service.
+- An error occurs when connecting SPS via the Power BI Service.
```
Comment (c4tc4t) [inline SafeguardSessions/TROUBLESHOOTING.md:9]: `to SPS`

File: SafeguardSessions/TROUBLESHOOTING.md [hunk 2]
```diff
@@ -2,25 +2,25 @@
...
+To see if there were any errors during the data import, you can check the **Info** table. When it indicates an error, you can read about the specific error by searching for the error code in the [Errors raised by the Power BI Connector] section of this document. The **Sessions** table will contain the exact error record that you can inspect in the [Data] pane of [Power Query Editor].
 
 #### A pop-up appears indicating some type of error.
 
 If the error is handled by the Power BI Connector, the pop-up will contain the error code and error message what you can find in [Errors raised by the Power BI Connector] section to learn more about the error.
 
-#### An error occurs when connecting via the Power BI Service.
+#### An error occurs when connecting SPS via the Power BI Service.
```
Comment (c4tc4t) [inline SafeguardSessions/TROUBLESHOOTING.md:21]: `to SPS`

**Review by c4tc4t — CHANGES_REQUESTED** [review]
Thanks for the updates! :-)
Two minor suggestions in TROUBLESHOOTING.md, otherwise looks good to me. :-)
Thank you!

File: SafeguardSessions/TROUBLESHOOTING.md (hunk 1 above)
Comment (szszeker) [reply SafeguardSessions/TROUBLESHOOTING.md]: Done!

File: SafeguardSessions/TROUBLESHOOTING.md (hunk 2 above)
Comment (szszeker) [reply SafeguardSessions/TROUBLESHOOTING.md]: Done!

**Review by davedusa — APPROVED** [review]

**Review by marcellmeszaros — APPROVED** [review]

**Pull request merged**